include/dali/core/tensor_view.h

```diff
@@ -650,6 +666,27 @@ TensorListView<StorageBackend, T, ndim> make_tensor_list(T *const *data,
   return { data, std::move(shape) };
 }
 
+template <typename StorageBackend, int ndim, typename T>
+TensorListView<StorageBackend, T, ndim> make_tensor_list(
+    const TensorView<StorageBackend, T, ndim> &tv) {
+  return {tv.data, {tv.shape}};
+}
```

Comment:
This overload seems a bit off. Where do you need it? Can you perhaps use:
```make_tensor_list({tv});```
?

Reply:
My feeling is that TensorView might successfully be a 1-sample TensorListView. Therefore I've put the function. I guess using `make_tensor_list({tv});` does not hurt, but probably `make_tensor_list(tv);` is more obvious.

@banasraf , @awolant , how do you feel about this overload?

Reply:
For me it looks like it strips the outer dimension and converts it to the batch.
NCHW -> slice across N and produce N x CHW tensor.

Reply:
Added documentation to explain what this function does.